test/packaging/rpm/explicit-target.py

```diff
@@ -77,7 +77,7 @@
 
 expect = """
 scons: *** Setting target is not supported for rpm.
-""" + test.python_file_line(test.workpath('SConstruct'), 23)
+""" + test.python_file_line(test.workpath('SConstruct'), 12)
```

Comment:
I've fought this before: we are going to need a different solution for this, as it seems the way Python reports the line number changed at 3.8, from last line to first line - we're never going to be able to match it accurately across that boundary.
3.7:
```============ BEGIN STDERR (len=123)

scons: *** Setting target is not supported for rpm.
File "/tmp/testcmd.1120038.thh9kg5h/SConstruct", line 23, in <module>
```
3.8:
```
============ BEGIN STDERR (len=123)

scons: *** Setting target is not supported for rpm.
File "/tmp/testcmd.1120287.54aen223/SConstruct", line 12, in <module>
```